File: WebUI/ClientApp/src/app/core/entity/account/account.ts
import { Token } from "../token/token";

export class Account {
  public id: number;
  public email: string;
  public password: string;
  public createAt: Date;
  public modiftAt: Date;

  public tokenId: number;
  public token: Token = new Token();

  constructor() { }
}

File: Domain/Entities/Account.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Entities
{
    public class Account
    {
        public int Id { get; set; }
        public string Email { get; set; }
        public string Password { get; set; }
        public DateTime CreateAt { get; set; }
        public DateTime ModifyAt { get; set; }

        public int? TokenId { get; set; }
        public Token Token { get; set; }

        public int? LocationId { get; set; }
        public Location Location { get; set; }

        public Court Court { get; set; }
        public Customer Customer { get; set; }
    }
}
